mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubClientService.kt

```diff
@@ -37,11 +38,16 @@ import kotlin.jvm.Throws
 public interface GitHubClientService {
 
     /**
-     * Searches for user `Mentions` by the GitHub name of the user.
+     * Searches for user `Mention`s by the GitHub name of the user. The mentions that occurred
```

Comment:
Can we please _always_ aim towards having the same structure for the documentation pieces.

1. First paragraph is a single sentence. It tells what this "thing" is, or what it does.
2. The rest of the documentation expands the idea. It's a plain text, split into one or more paragraphs.